src/helpers/parser/listItemParser.js

```diff
@@ -270,13 +269,18 @@ const parseConsulData = (data, query, skipLastDivider) => {
         momentFormatUtcToLocal(consulData.endsAt);
     }
 
+    if (query === QUERY_TYPES.CONSUL.PUBLIC_PROPOSALS && !consulData.published) {
+      title = `${texts.consul.draft} - ${consulData.title ?? consulData.body}`;
```

Comment:
as `title` is already set to `consulData.title ?? consulData.body` you can use the `title` here: 

```js
title = `${texts.consul.draft} - ${title}`
```

Reply:
I don't understand how I missed this. 🤦‍♂️

Reply:
:) happens